### ISSUE TITLE
[SAP][Button]: On mouse hover button border fails to meet color contrast ratio of 3:1

### ISSUE THREAD

**Issue by rama1990br** [SAP/ui5-webcomponents-react#4439]
### Describe the bug

On mouse hover button border fails to meet color contrast ratio of 3:1.

### Reproduction steps

**Instance:**
https://qacand.hcm.ondemand.com/sf/scriptintegrationsettings?company=ESSRIOLGA&username=jkincaid&password=Welcome

**Steps:**
Login to instance.
Directly goto link given
Script-Based Integration Settings page is opened. Goto Add Vendor button
Verify color on mouse hover.
Then on Script-Based Integration Settings, verify color for all button/UI elements on mouse hover

**Actual Results:**
On mouse hover, buttons have light grey border with color ratio of 1.18 which does not meet 3:1 ratio

### Expected Behaviour

The text and border foreground color and background color of each interactive element should have a contrast ratio of at least 3:1. 

### Screenshots or Videos

![info icon border color issue](https://user-images.githubusercontent.com/32254036/229024881-7b58ecd9-9c96-4746-8262-2d15896ba5dd.png)
![border color ratio issue](https://user-images.githubusercontent.com/32254036/229024883-2d6e8138-4c93-4f4f-b1da-4f48e9e20906.png)

### UI5 Web Components for React Version

1.5.1

### UI5 Web Components Version

1.9.1

### Browser

Chrome, Edge, Firefox, Safari

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Hi @rama1990br,

please do not post internal system links with their credentials in the issue description. These issues are public and everybody can use these credentials to access our internal test systems. I've now removed the information from the issue and moved them to internal slack channel.

This is also explicitly mentioned in our issue template:
> Please don't disclose any internal or sensitive information like: internal urls, login credentials, screenshots of applications or features in development.
This is a public GitHub repository and all the information is publicly available.

As the affected component is developed in the UI5 Web Components repository, I'll forward this issue to our colleagues.

Best regards,
Marcus
